data/json/items/containers.json

```diff
@@ -3329,5 +3329,57 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ] ],
     "flags": [ "COLLAPSE_CONTENTS" ]
+  },
+  {
+    "id": "lunchbox",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "lunchbox", "str_pl": "lunchboxes" },
+    "description": "A plastic box, commonly used to store lunch and utensils to eat it with.",
+    "weight": "282 g",
+    "volume": "2050 ml",
+    "longest_side": "21 cm",
+    "price": 1400,
+    "price_postapoc": 100,
+    "material": [ "plastic" ],
+    "symbol": "l",
+    "color": "green",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": false,
+        "open_container": false,
+        "max_contains_volume": "1100 ml",
+        "max_contains_weight": "2 kg"
+      }
+    ],
+    "qualities": [ [ "CONTAIN", 1 ] ],
+    "flags": [ "COLLAPSE_CONTENTS" ]
+  },
+  {
+    "id": "pencil_case",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "pencil case" },
+    "description": "A plastic box, commonly used to store lunch and utensils to eat it with.",
```

Comment:
The description is the same as the lunch box one, an oversight it seems.

Reply:
Ah yep, I copied it over as a template and forgot to change the description. Thanks for keeping an eye out! I'll fix it with the next commit.